app/Http/Controllers/Service/Admin/CentreUsersController.php

```diff
@@ -223,9 +223,7 @@ private function syncCentres($request, CentreUser $cu)
 
     public function download()
     {
-        $workers = CentreUser::get()->sortBy(function($centreUser) {
-            return sprintf('%-12s%s', $centreUser->homeCentre->name, $centreUser->name);
-        });
+        $workers = CentreUser::get()->sortBy('name')->sortBy('homeCentre.name');
```

Comment:
as per https://github.com/laravel/ideas/issues/11#issuecomment-329866327 ?
give it a go.

Reply:
yes, it's similar to that but then there's another comment after [https://github.com/laravel/ideas/issues/11#issuecomment-329970458](https://github.com/laravel/ideas/issues/11#issuecomment-329970458) that says that the reverse sorting works accidentally and is based on undefined behaviour. I saw it it here, too [https://medium.com/@codebyjeff/the-keys-are-the-key-in-laravel-multiple-field-sortby-60cd5bf3f78b](https://medium.com/@codebyjeff/the-keys-are-the-key-in-laravel-multiple-field-sortby-60cd5bf3f78b)

so I don't know which one to choose... thoughts?